package.json

```diff
@@ -1,33 +1,36 @@
 {
   "name": "Queensland-Government-Web-Template",
-  "version": "4.0.13",
+  "version": "4.0.14",
   "description": "This template is designed to provide a template for all Franchise websites, and the underpinning technology for new Agency websites.",
   "main": "gulpfile.babel.js",
   "repository": {
     "type": "git",
-    "url": "https://github.com/qld-gov-au/qg-web-template.git"
+    "url": "git+https://github.com/qld-gov-au/qg-web-template.git"
   },
   "scripts": {
     "clean": "gulp clean-build && gulp clean-release",
     "prebuild": "gulp clean-build",
     "build": "gulp build",
     "test": "jest",
+    "test-ci": "npm run watch & (echo \"sleeping 10\"; sleep 10;) && npm run test",
     "postbuild": "gulp clean-redundant-build",
     "release": "gulp clean-release && export NODE_ENV=prod && npm run build && node gulp/build-tasks/node-docs-flatten.js && gulp release && node gulp/release-tasks/node-template-flatten.js && gulp replace-links",
     "postrelease": "gulp clean-redundant-release",
     "start-legacy": "npm run build && gulp watch",
     "start": "npm run build && concurrently \"gulp watch\" \"npm:storybook\"",
     "watch": "gulp watch",
     "gh-pages-legacy": "export NODE_ENV=prod && npm run release && node gulp/gh-pages.js",
-    "gh-pages": "export PUBLIC_PATH=https://qld-gov-au.github.io/qg-web-template && npm run build-storybook && node gulp/gh-pages.js && echo 'publishing complete.'",
+    "gh-pages": "npm run build-storybook-gh-pages && npm run build-storybook && node gulp/gh-pages.js && echo 'publishing complete.'",
     "publish-npm": "export NODE_ENV=prod && npm run release && gulp wt-clean && gulp wt-clone && gulp wt-sync && cd web-template-release && git add . && git commit -m ${npm_package_version} && npm login && np ${npm_package_version}",
     "publish-test-npm": "npm run release && export NODE_ENV=test &&  gulp wt-clean && gulp wt-clone && gulp wt-sync && gulp wt-branch && gulp wt-updateVersion && gulp wt-add && gulp wt-commit && gulp wt-push",
     "create-swe-release": "export NODE_ENV=prod && gulp swe-add && gulp swe-tag && gulp swe-commit && gulp swe-push",
     "publish-cdn": "export NODE_ENV=prod && npm run release && gulp cdn-clean && gulp cdn-clone && gulp cdn-transfer && gulp cdn-add && gulp cdn-commit && gulp cdn-push",
     "publish-test-cdn": "npm run release && export NODE_ENV=test && gulp cdn-clean && gulp cdn-clone && gulp cdn-transfer && gulp cdn-branch && gulp cdn-add && gulp cdn-commit && gulp cdn-push",
     "postpublish": "gulp clean-publish",
     "storybook": "start-storybook -p 6006",
+    "storybook-ci": "start-storybook -p 6006 --ci",
     "build-storybook": "build-storybook",
+    "build-storybook-ci": "npm run storybook-ci & (echo \"sleeping 25\"; sleep 25;) && npm run build-storybook",
```

Comment:
Is there a way to detect when the task is done, instead of having a fixed-length sleep?

Reply:
the task `storybook-ci` is an on-going background job, i don't know a way to check it's initiated and trigger the following command